lib/util/pssh.js

```diff
@@ -124,6 +124,49 @@ shaka.util.Pssh = class {
     return psshBox;
   }
 
+  /**
+   * Creates a ClearKey cenc pssh blob from the given system ID and data.
```

Comment:
In fact, this only creates a ClearKey PSSH blob if the system ID is right.  It seems more that what this method does is create a v1 PSSH.

It's confusing, though, because you treat "data" as a single key ID, and leave the "data" payload of the PSSH box empty.

There are very few differences between this method and `createPssh`.  If you add version and key ID parameters to `createPssh`, you could make that method support v1 PSSHs.  Then, you could make a convenience method for ClearKey that provides the correct system ID, an empty data field, and passes along a list of key IDs.

Reply:
Thanks for the review. Your suggestion sounds good.
Sorry about the confusion regarding "data". 
Have made the changes. Please take a look.